api/v1_recommendations_gcp.go

```diff
@@ -31,3 +33,54 @@ func (svc *GcpRecommendationsV1) List() ([]RecommendationV1, error) {
 func (svc *GcpRecommendationsV1) Patch(recommendations RecommendationStateV1) (response RecommendationResponseV1, err error) {
 	return svc.client.Recommendations.patch(GcpRecommendation, recommendations)
 }
+
+// GetReport This is an experimental feature. Returned RecommendationID's are not guaranteed to be correct. Scoped to Lacework Account/Subaccount
+func (svc *GcpRecommendationsV1) GetReport(reportType string) (response []RecommendationV1, err error) {
+	gcpCfg, err := svc.client.Integrations.ListGcpCfg()
+	if err != nil {
+		return []RecommendationV1{}, err
+	}
+
+	// cis1.3 is off by default, so we won't be able to fetch previous reports.
+	
+	if len(gcpCfg.Data) == 0 {
+		return []RecommendationV1{}, errors.Wrap(err, "unable to find a GCP cloud account integration")
+	}
+
+	var projectID = gcpCfg.Data[0].Data.ID
+	var orgID = "n/a"
+
+	// TODO get all gcp projects for org
+	// cli/cmd/compliance_gcp.go:496
+	//if gcpCfg.Data[0].Data.IDType == "ORGANIZATION" {
+	//
+	//}
```

Comment:
what's up with this commented code and TODO?

Reply:
Will be removed be this Draft is moved to a full PR